states/corvee.mod.py

```diff
@@ -19,11 +19,7 @@ class State(BaseState):
 
     # module check function
     def check(self, _state):
-        now = datetime.now()
-        if now.weekday() == 4: # friday
-            return now.hour == 21 and now.minute == 0
-        elif now.weekday() == 5: # saturday
-            return now.hour == 12 and now.minute == 30
+        return len(self.names) > 0
```

Comment:
oh jeetje, `self.names` wordt nu niet ge-update, dus wordt hij nooit actief. Een State object wordt namelijk 1 keer aangemaakt, en alleen opnieuw gemaakt als de state gedraaid heeft (het is een Thread, en die kan je maar 1 keer starten).